service/history/api/signalworkflow/api.go

```diff
@@ -106,7 +106,8 @@ func Invoke(
 				request.GetSignalName(),
 				request.GetInput(),
 				request.GetIdentity(),
-				request.GetHeader()); err != nil {
+				request.GetHeader(),
+				request.GetSkipGenerateWorkflowTask()); err != nil {
```

Comment:
also need to update the `createWorkflowTask` I think. Can you plz double check the logic for signal with start workflow? I think some update is also needed there.

Please also add some tests for those two apis re. skipping workflow task generation.

Reply:
On the signal with start, I did update `signal_with_start_workflow#signalWorkflow()` to pass through the new flag. Is there some other change that is also required?

Could you elaborate more on what change is needed for `createWorkflowTask`? I'm not sure I understand what change is needed there.